debian/rules

```diff
@@ -37,7 +37,6 @@ DEB_DH_SYSTEMD_START_ARGS_tarantool-common  := --no-restart-on-upgrade tarantool
 
 # Needed for proper backtraces in fiber.info()
 DEB_DH_STRIP_ARGS	        := -X/usr/bin/tarantool
-export DEB_BUILD_MAINT_OPTIONS = hardening=-stackprotector
```

Comment:
Don't know, which preprocessor flags are actually passed on the hardened Debian's build (is it default one?), but I found a recommendation to take care to `CPPFLAGS`:

* https://wiki.debian.org/HardeningWalkthrough#buildsystems_using_cmake
* https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=653916#10

Maybe it worth to look around here a bit.

Reply:
In this patch all flags are passed via CFLAGS/CXXFLAGS/LDFLAGS, without using CPPFLAGS.

Reply:
I guess this recommendation exists, because Debian's build system may pass CPPFLAGS. If the project don't take it, some security flags will be ignored in the Debian/Ubuntu package build. Just my guess.

Reply:
> Debhelper (since 0.9.20120417, only with compat=9 and dh_auto* commands!) and cdbs (since 0.4.110) handle this automatically so the workaround is no longer necessary if they are used.

(From the first link above.)

Hm, I missed this note somehow. It seems, the problem is worked around now on the Debian build system side.